examples/qualcomm/scripts/mobilenet_v2.py

```diff
@@ -102,6 +104,18 @@ def get_data_loader():
         type=str,
         required=True,
     )
+    parser.add_argument(
+        "--ip",
+        help="IPC address for delivering execution result",
+        default="",
+        type=str,
+    )
+    parser.add_argument(
+        "--port",
+        help="IPC port for delivering execution result",
+        default=-1,
+        type=int,
+    )
```

Comment:
nit: If you refactor your examples you can consolidate all the boiler plate code.

Also if you can add a comment on how is this generally useful, it would be good

Reply:
Thank you for this suggestion. We will gradually refactor this in following PR especially for incoming backend support.